preprocessing/nextclade/tests/factory_methods.py

```diff
@@ -12,23 +12,31 @@
 
 
 @dataclass
-class TestCase:
+class ProcessingTestCase:
     name: str
     input: UnprocessedEntry
     expected_output: ProcessedEntry
 
+
 @dataclass
 class UnprocessedEntryFactory:
     _counter: int = 0
 
+    @classmethod
+    def reset_counter(cls):
+        cls._counter = 0
+
     @staticmethod
     def create_unprocessed_entry(
         metadata_dict: dict[str, str],
+        accession: str | None = None,
```

Comment:
Maybe this would actually be better named accession_id - as here we take the id add the prefix LOC_ to create an accession.

Whereas create_processed_entry actually takes the accession.

Reply:
added